openedx/core/djangoapps/edx_global_analytics/tasks.py

```diff
@@ -3,21 +3,91 @@
 and send this data to appropriate service for further processing.
 """
 
-import datetime
 import json
 
 import requests
 from celery.task import task
 
 from django.conf import settings
 from django.contrib.sites.models import Site
-from django.db.models import Count
-from django.db.models import Q
 
 from xmodule.modulestore.django import modulestore
-from student.models import UserProfile
 from .models import TokenStorage
 
+from .utils import calculate_active_students_amount, calculate_students_per_country, \
+                    get_previous_day_start_and_end_dates, get_previous_week_start_and_end_dates, \
+                    get_previous_month_start_and_end_dates
+
+
+def calculate_instance_information():
```

Comment:
I would suggest renaming this method to smth like `get_instance_information` or `fetch_instance_information`